Seminars/Sem.01/Pract.01/Task04.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+
+using std::cin;
+using std::cout;
+using std::endl;
+
+const int MAX_NAME_SIZE = 30;
+
+void copyStr(char* copy, const char* arr)
+{
+	int count = 0;
+
+	while (arr[count] != '\0')
+	{
+		copy[count] = arr[count];
+		count++;
+	}
+
+	copy[count] = '\0';
+}
+
+int getStrLength(const char* str)
+{
+	int count = 0;
+
+	while (str[count++] != '\0')
```

Comment:
Would increment here due to readability, even if we do , move the brackets to the same line.